build.gradle.kts

```diff
@@ -160,3 +161,26 @@ tasks.register<JavaExec>("ktlintFormat") {
     main = "com.pinterest.ktlint.Main"
     args = listOf("-F") + lintPaths
 }
+
+apiValidation {
+    nonPublicMarkers += "aws.smithy.kotlin.runtime.InternalApi"
+
+    ignoredProjects += setOf(
```

Comment:
We can probably ignore `smithy-kotlin-codegen` for now as well since we aren't making any guarantees on the API stability for the code generator